proseco/core.py

```diff
@@ -507,7 +507,8 @@ def plot(self, ax=None, **kwargs):
         kwargs.setdefault('bad_stars', np.zeros(len(kwargs['stars']), dtype=bool))
 
         fig = plot_stars(attitude=self.att, ax=ax, **kwargs)
-        plt.show()
+        if 'agg' not in plt.get_backend().lower():
+            plt.show()
```

Comment:
In shiny (new matplotlib) one gets a warning in tests about calling `plt.show()` for a non-interactive backend.